src/SetCookie.php

```diff
@@ -168,11 +176,11 @@ public function toHeaderValue() : string
             );
         }
 
-        if (empty($this->path) === false) {
+        if ($this->path !== '') {
```

Comment:
why would someone pass `0` as variable? I feel it's something the caller should take care of. To ensure it has the right value.